pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -337,6 +339,26 @@ func Newk8sOrchestrator(ctx context.Context, controllerClusterFlavor cnstypes.Cn
 	return k8sOrchestratorInstance, nil
 }
 
+func getReleasedVanillaFSS() map[string]string {
+	return map[string]string{
+		common.CSIMigration:                   "true",
+		common.OnlineVolumeExtend:             "true",
+		common.TriggerCsiFullSync:             "false",
+		common.AsyncQueryVolume:               "true",
+		common.BlockVolumeSnapshot:            "true",
+		common.CSIWindowsSupport:              "true",
+		common.ListVolumes:                    "true",
+		common.PVtoBackingDiskObjectIdMapping: "false",
```

Comment:
Remove `PVtoBackingDiskObjectIdMapping` as well. Anyone willing to try this feature out must not be blocked as we hardcoded the value.

Reply:
Removed.